format-common/src/main/java/io/cdap/plugin/format/input/PathTrackingInputFormatProvider.java

```diff
@@ -54,17 +56,42 @@ public Map<String, String> getInputFormatConfiguration() {
   }
 
   /**
-   * Perform validation on the provided configuration.
-   *
-   * Deprecated since 2.3.0. Use {@link PathTrackingInputFormatProvider#validate(FormatContext)} method instead.
-   */
+  * Perform validation on the provided configuration.
+  *
+  * Deprecated since 2.3.0. Use {@link PathTrackingInputFormatProvider#validate(FormatContext)} method instead.
+  */
   @Deprecated
   protected void validate() {
-    // no-op
+  // no-op
```

Comment:
fix indentation. Please make sure your IDE settings are correct, this has been a problem in every round of reviews.